app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/repository/di/AuthenticatorBridgeModule.kt

```diff
@@ -33,10 +38,21 @@ object AuthenticatorBridgeModule {
     fun provideAuthenticatorBridgeManager(
         factory: AuthenticatorBridgeFactory,
         symmetricKeyStorageProvider: SymmetricKeyStorageProvider,
-    ): AuthenticatorBridgeManager = factory.getAuthenticatorBridgeManager(
-        connectionType = AuthenticatorBridgeConnectionType.DEV,
-        symmetricKeyStorageProvider = symmetricKeyStorageProvider,
-    )
+        featureFlagManager: FeatureFlagManager,
+    ): AuthenticatorBridgeManager =
+        if (featureFlagManager.getFeatureFlag(LocalFeatureFlag.PasswordManagerSync)) {
+            factory.getAuthenticatorBridgeManager(
+                connectionType = AuthenticatorBridgeConnectionType.DEV,
+                symmetricKeyStorageProvider = symmetricKeyStorageProvider,
+            )
+        } else {
+            // If feature flag is not enabled, return noop bridge manager so we never
+            // connect to bridge service:
+            object : AuthenticatorBridgeManager {
+                override val accountSyncStateFlow: StateFlow<AccountSyncState>
+                    get() = MutableStateFlow(AccountSyncState.Loading)
```

Comment:
Good question. Nope, this value will never be consumed. The main goal here is just to NOT instantiate the `AuthenticatorBridgeManagerImpl`, since that will do stuff in `init` that we don't wanna do if the feature flag is off.